man/systemd-coredump.xml

```diff
@@ -192,6 +192,17 @@ COREDUMP_FILENAME=/var/lib/systemd/coredump/core.Web….552351.….zst
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>COREDUMP_BY_PIDFD=</varname></term>
+        <listitem><para>If the crashed process was analyzed using a PIDFD provided by the kernel (requires
+        kernel v6.16) then this field will be present and set to <literal>1</literal>. If this field is
+        not set or <literal>0</literal>, then the crashed process was analyzed via a PID, which is known
```

Comment:
I think "not set or" should be dropped here. We describe the code as of the current version, which we'll always set the property. It'll be not set for older versions of systemd, but this is true for anything that we add, and this information is provided by the "Added in version 258" string below.

Reply:
As per Lennart's request this is also skipped on newer systemd, if the core is forwarded to a container guest instead of being stored on the host, so I think it should stay. But it can't be set to '0' anymore, so I'll drop that part.